src/core/languages.js

```diff
@@ -50,15 +50,6 @@ export const unfilteredLanguages = {
     English: 'Bambara',
     native: 'Bamanankan',
   },
-  'bn-BD': {
-    English: 'Bengali (Bangladesh)',
-    native:
-      '\u09ac\u09be\u0982\u09b2\u09be (\u09ac\u09be\u0982\u09b2\u09be\u09a6\u09c7\u09b6)',
-  },
-  'bn-IN': {
-    English: 'Bengali (India)',
-    native: '\u09ac\u09be\u0982\u09b2\u09be (\u09ad\u09be\u09b0\u09a4)',
-  },
```

Comment:
I don't think this should be removed.

Reply:
@willdurand I can delete it but i guess they are not gonna use in future then why we need to add them here?

Reply:
I have no strong opinion about it but I'd say we might still need that in the future.

Reply:
Ok updating no worries :)

Reply:
Hmm 👀 , updating the patch :)